features/shop/cart_with_different_taxes_for_different_zones.feature

```diff
@@ -0,0 +1,50 @@
+@ui-cart
+Feature: Cart with different tax rates for different time zones
+    In order to order products with different tax rate for different zones
+    As a Customer
```

Comment:
Yes, but I think we should differentiate between someone who is just visiting and someone who started a checkout, we have much more info about him etc.